src/components/ebay-breadcrumb/index.js

```diff
@@ -0,0 +1,60 @@
+const markoWidgets = require('marko-widgets');
+const emitAndFire = require('../../common/emit-and-fire');
+const processHtmlAttributes = require('../../common/html-attributes');
+const template = require('./template.marko');
+
+const constants = {
```

Comment:
Each of these constants is only used a single time, and doesn't really improve clarity, so I would skip this `constants` object altogether.

Reply:
I agree with this.